app/[username]/[game]/[run]/run.tsx

```diff
@@ -61,18 +59,20 @@ export default function RunDetail({
     const forceRealTime = !!globalGameData.forceRealTime;
 
     const [useGameTime, setUseGameTime] = useState(
-        run.hasGameTime &&
+        !!(
+            run.hasGameTime &&
             !!run.gameTimeData &&
             run.gameTimeData.personalBest &&
             parseInt(run.gameTimeData.personalBest) &&
```

Comment:
Is the first one even needed since it'd be null and due to it being NaN anyways, it wouldn't be true?

```suggestion
            parseInt(run.gameTimeData.personalBest) &&
```